components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/UpdateAdvancedConfigurations.java

```diff
@@ -71,4 +78,14 @@ private void updateCertificate(Certificate certificate, ServiceProvider serviceP
             }
         }
     }
+
+
+    private void handleAdditionalSpProperties(List<AdditionalSpProperty> spAdditionalProperties) {
+
+        // Support is not there only for POST or PATCH yet, Only for GET
+        if (!CollectionUtils.isEmpty(spAdditionalProperties)) {
+            throw buildNotImplementedError(ADDITIONAL_SP_PROP_NOT_IMPLEMENTED.getCode(),
```

Comment:
If additional properties are empty, the expectation would be to send an empty list, instead of not implemented.

Reply:
We literally do not support **additional properties** and we are checking if the list is not empty to throw not-implemented. What is your suggestion on that?